client/src/sections/quests/Level6.tsx

```diff
@@ -0,0 +1,92 @@
+import { $location } from "libram";
+import { List, ListIcon, ListItem } from "@chakra-ui/react";
+import Chevrons from "../../components/Chevrons";
+import Line from "../../components/Line";
+import QuestTile from "../../components/QuestTile";
+import { useMyHash, useNumericModifier } from "../../hooks/useCall";
+import useHave from "../../hooks/useHave";
+import { atStep, Step, useQuestStep } from "../../hooks/useQuest";
+import { $skill } from "../../util/makeValue";
+
+const Level6: React.FC = () => {
+  const step = useQuestStep("questL06Friar");
+  const myHash = useMyHash();
```

Comment:
can you revise this to use the new style (i.e. no "use" / hooks)?

Reply:
Been a hot minute since I've really dug into yorick but I think we're good now